components/BaseRadioButton/styles.js

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+
+export const Radio = styled.label`
+  color: #475065;
+  display: grid;
+  grid-gap: 0.5rem;
+  grid-template-columns: min-content auto;
+  margin-bottom: 1rem;
+`;
+export const RadioInput = styled.span`
+  display: flex;
+`;
+export const RadioLabel = styled.span`
+  color: #475065;
+  font-size: 0.875rem;
+  line-height: 1.375rem;
+`;
+export const RadioControl = styled.span`
+  border-radius: 50%;
+  border: 0.125rem solid currentColor;
+  color: #ccc;
+  display: block;
+  height: 1.5rem;
+  transform: translateY(0.06rem);
+  width: 1.5rem;
+`;
+export const Input = styled.input`
```

Comment:
```suggestion
export const Input = styled('input').attrs({ type: 'radio' })`
```
Sería cool que se llame RadioInput pero el nombre ya está tomado arriba, y me pregunto por qué necesitaría existir el otro `RadioInput` si es solo un wrapper que agrega flex, pienso que puede ser por qué el padre es grid, pero no sé, hay una explicación?

Reply:
Es mas que nada para entender que RadioInput esta basado en Input y el RadioControl, y que el input se esta ocultando 🙌